docs/metrics/prometheus.md

```diff
@@ -5,7 +5,7 @@ title: "Prometheus Example"
 
 ## The ZMX Prometheus example
 
-```scala mdoc:invisible
+```scala
```

Comment:
Would suggest keeping `mdoc` everywhere so things don't get out of date.

Reply:
Definitely - Have removed it from a couple of places for now just to complete the rendering process. This having said, in some places the code block will be just documentation and cant be compiled. In those cases i will remove the mdoc